inference-engine/thirdparty/CMakeLists.txt

```diff
@@ -53,12 +53,28 @@ function(build_with_lto)
 
     target_include_directories(pugixml INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/pugixml/src>")
 
+    add_library(half INTERFACE)
```

Comment:
Do we add a new third party library only for template_plugin?

Reply:
* I have another plugin that can use it. 
* I can move it to third party directory inside plugin folder
* I believe that all host type conversions should be done with GAPI preprocessing here. But it is not implemented yet.

Reply:
Removed

Reply:
@ilyachur  BTW Do we have ability to convert fp16 ngraph::function into fp32 as it is done in CPU plugin but using ngrap only?

Reply:
@apankratovantonp At the current moment we don't have transformation pass to convert fp16 to fp32... 
Some time ago we added only path for fp32 to fp16 conversion (ngraph/pass/convert_fp32_to_fp16.hpp)

Reply:
`half` library is removed